examples/config.hocon.pubsub.minimal

```diff
@@ -0,0 +1,21 @@
+collector {
+  interface = "0.0.0.0"
+  port = 80
+
+  streams {
+    good = "{{good}}"
+    bad = "{{bad}}"
+
+    sink {
+      enabled = google-pub-sub
+      googleProjectId = "{{googleProjectId}}"
+
+      backoffPolicy {
```

Comment:
These are good defaults:

```
backoffPolicy {
        minBackoff = 1000
        maxBackoff = 1000
        totalBackoff = 10000
        multiplier = 1
      }
```

Reply:
There is a conflict between good defaults for SQS/Kinesis and GCP.
Would it be a good idea to put `application.conf` in each component instead of the core to resolve this?

Reply:
Yes, I think it would work the same way as @istreeter suggested for the `enabled` default: an `application.conf` in each module that only enables that particular sink.